PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-storage/src/main/java/com/microsoft/azure/toolkit/intellij/storage/component/AccountNameTextField.java

```diff
@@ -51,7 +51,7 @@ private AzureValidationInfo doValidateValue() {
         // validate availability
         CheckNameAvailabilityResultEntity resultEntity;
         try {
-            resultEntity = Azure.az(AzureStorageAccount.class).checkNameAvailability(subscriptionId, this.getValue());
+            resultEntity = Azure.az(AzureStorageAccount.class).forSubscription(subscriptionId).checkNameAvailability(this.getValue());
```

Comment:
Name for storage account/redis should be global, shall we move it to ResourceManager?